.github/merge_rules.json

```diff
@@ -273,6 +273,19 @@
          "deploy-linux-xenial-cuda11.3-py3.7-gcc7 / build"
       ]
    },
+   {
+      "name": "MPS",
+      "patterns": [
+         "test/test_mps.py",
+         "aten/src/ATen/mps/**",
+         "aten/src/ATen/native/mps/**"
+      ],
+      "approved_by": ["kulinseth"],
+      "mandatory_checks_name": [
+         "Facebook CLA Check",
+         "Lint"
```

Comment:
I'm guessing we would want to add an MPS build here once an MPS build can be added to CI?

Reply:
We don't want to force `ciflow/trunk` on all MPS PRs, but they are affecting only MacOS builds